lib/cdo/properties.rb

```diff
@@ -36,128 +36,128 @@ def self.delete(key)
+      'Birmingham' => 815_304,
+      'Madison' => 814_780,
     },
     # Generated by `fetch_hoc_metrics['countries'].first(40).to_h`. Note that we
     # take the top forty, as that many countries are shown on the leaderboard.
     'countries' => {
-      'United States' => 176_408_221,
-      'United Kingdom' => 17_041_126,
-      'Other' => 10_726_283,
-      'Canada' => 5_907_322,
-      'Australia' => 5_184_491,
-      'Turkey' => 3_425_775,
-      'France' => 3_243_916,
-      'Italy' => 3_191_676,
-      'Taiwan' => 2_658_012,
-      'Brazil' => 2_493_510,
-      'Spain' => 2_449_625,
-      'Poland' => 2_055_855,
-      'India' => 1_959_610,
+      'United States' => 269_119_961,
+      'United Kingdom' => 23_640_611,
+      'Other' => 10_759_918,
+      'Canada' => 9_938_588,
+      'Australia' => 7_795_086,
+      'Turkey' => 5_320_296,
+      'France' => 5_066_582,
+      'Italy' => 4_849_916,
+      'Taiwan' => 3_824_526,
+      'Spain' => 3_568_500,
+      'Poland' => 3_386_767,
+      'Brazil' => 3_125_785,
+      'Mexico' => 2_478_498,
+      'India' => 2_317_183,
+      'China' => 2_065_029,
+      'Republic of Korea' => 2_021_896,
       'Korea, Republic of' => 1_883_003,
-      'Mexico' => 1_824_703,
-      'Ireland' => 1_653_915,
-      'Greece' => 1_363_365,
-      'Colombia' => 1_329_756,
-      'China' => 1_267_930,
-      'Romania' => 1_221_432,
-      'Japan' => 1_199_082,
-      'Netherlands' => 1_079_824,
-      'Finland' => 1_065_716,
-      'New Zealand' => 987_037,
-      'Germany' => 942_334,
-      'Sweden' => 941_164,
-      'Russian Federation' => 908_392,
-      'United Arab Emirates' => 893_042,
-      'Republic of Korea' => 886_765,
-      'Israel' => 859_384,
-      'Vietnam' => 826_898,
-      'Ukraine' => 804_945,
-      'Argentina' => 671_853,
-      'Egypt' => 591_940,
-      'Denmark' => 565_908,
-      'Thailand' => 558_549,
-      'Malaysia' => 539_057,
-      'Portugal' => 526_543,
-      'Hungary' => 498_396,
-      'Chile' => 491_365
+      'Ireland' => 1_786_424,
+      'Greece' => 1_736_384,
+      'Japan' => 1_703_289,
+      'Romania' => 1_678_404,
+      'Colombia' => 1_587_535,
+      'Israel' => 1_441_735,
+      'Netherlands' => 1_438_695,
+      'Finland' => 1_417_359,
+      'New Zealand' => 1_397_405,
+      'Sweden' => 1_390_016,
+      'Germany' => 1_386_411,
+      'Ukraine' => 1_288_207,
+      'United Arab Emirates' => 1_282_278,
+      'Vietnam' => 1_180_923,
+      'Russian Federation' => 908_395,
+      'Norway' => 819_309,
+      'Argentina' => 801_222,
+      'Denmark' => 801_015,
+      'Thailand' => 765_155,
+      'Portugal' => 694_856,
+      'Malaysia' => 664_502,
+      'Egypt' => 657_018,
+      'Belgium' => 644_326
     },
-    # The count was reset to 0 in June 2016 as the result of moving to HOC2016
-    # from HOC2015.
+    # The count was reset to 0 in June 2017 as the result of moving to HOC2017
+    # from HOC2016.
     'total_hoc_count' => 0,
```

Comment:
Note that we have 255 events for HOC 2017 in the DB at this time. That said, I'm going to "ignore" them in this update.